lib/matplotlib/axes/_axes.py

```diff
@@ -4111,21 +4113,26 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
         *xscale*: [ 'linear' | 'log' ]
            Use a linear or log10 scale on the horizontal axis.
 
-        *scale*: [ 'linear' | 'log' ]
+        *yscale*: [ 'linear' | 'log' ]
            Use a linear or log10 scale on the vertical axis.
 
         *mincnt*: [ *None* | a positive integer ]
            If not *None*, only display cells with more than *mincnt*
            number of points in the cell
 
         *marginals*: [ *True* | *False* ]
-           if marginals is *True*, plot the marginal density as
+           If marginals is *True*, plot the marginal density as
            colormapped rectagles along the bottom of the x-axis and
-           left of the y-axis
+           left of the y-axis.
 
         *extent*: [ *None* | scalars (left, right, bottom, top) ]
            The limits of the bins. The default assigns the limits
-           based on gridsize, x, y, xscale and yscale.
+           based on *gridsize*, *x*, *y*, *xscale* and *yscale*.
+
+           If *xscale* or *yscale* is set to 'log', the limit is
+           expected as a power of 10. E.g. for x-limits of 1 and 
```

Comment:
:bike: :house:  Should this be '..the limits are the power to raise 10 to to compute the range of the bins.'?  I would read 'expected as a power of 10' to mean the inputs must be of the form [10, 100, 1000, ...].